.gitlab-ci.yml

```diff
@@ -6,7 +6,7 @@ stages:
   - deploy-extended
 
 variables:
-  KUBESPRAY_VERSION: v2.26.0
+  KUBESPRAY_VERSION: v2.27.0
```

Comment:
Ah I see the problem. This should in fact stay at 2.26.0 on master:
See https://github.com/kubernetes-sigs/kubespray/blob/b4768cfa9137b9e85fc786b6c5b0075d93ac2edb/RELEASE.md?plain=1#L15

(KUBESPRAY_VERSION really mean : the tag against which we test upgrade, which is more or less the last released tag on master, but not on relase branch (where it would be the second last tag in that release branch AND in master).

It's pretty confusing. I tried a while ago to automate computing from git information, but we just settled on doing it this way because there is other stuff to do ^)

Reply:
I think I'll wait until after the 2.27 release to make this change, and then I'll close this PR later.